packages/golang-client/src/index.ts

```
@@ -34,7 +34,11 @@ const gofmt = (code: string) => {
 	} catch (e: any) {
 		// we silently ignore the error on purpose
 		// It's not a must to prettify the code
-		console.error('gofmt is not installed. If you want to prettify the generated code, please install gofmt');
+		if (e instanceof Error && e.message.indexOf('ENOENT') >= 0) {
```

Swallowing was intentional because we can't expect that everyone has go installed.

---

The problem was that it was also swallowing  other errors, like feeding gofmt invalid code. With this change, it will still swallow the exception when gofmt is not present (printing a warning), but it will fail when gofmt is installed but returns an error.

---

Thanks for the explanation. Makes sense.